macros/create_invocations_table_if_not_exists.sql

```diff
@@ -56,8 +56,8 @@
         dbt_cloud_run_id STRING,
         dbt_cloud_run_reason_category STRING,
         dbt_cloud_run_reason STRING,
-        env_vars ARRAY,
-        dbt_vars ARRAY
+        env_vars OBJECT,
+        dbt_vars OBJECT
```

Comment:
Figured OBJECT datatype would make more sense than ARRAY as these two columns will always be as a dict structure. But would VARIANT datatype make more sense?

Reply:
Object is most explicit so makes sense here